app/ts/components/App.tsx

```diff
@@ -173,12 +152,17 @@ export function App() {
 				<p class = 'sub-title'>Make immutable ENS domains and subdomains</p>
 			</div>
 		
-			<input 
-				class = 'input' 
-				type = 'text' 
-				placeholder = '2.horswap.eth' 
-				value = { inputValue.value } 
-				onInput = { e => handleInput(e.currentTarget.value) }
+			<DebounceInput
+				placeholder = '2.horswap.eth'
+				delay = { 1000 }
+				onDebouncedChange = { () => {
+					console.log('update!')
+					updateInfos(true)
+				} }
```

Comment:
Remove debug logging?  If you do you can simplify to:
```suggestion
				onDebouncedChange = { () => updateInfos(true) }
```